configuration.php

```diff
@@ -6,6 +6,7 @@
 $this->providePermission('grafana/graphconfig', $this->translate('Allow to configure graphs.'));
 $this->providePermission('grafana/debug', $this->translate('Allow to see module debug informations.'));
 $this->providePermission('grafana/showall', $this->translate('Allow access to see all graphs of a host.'));
+$this->providePermission('grafana/enableLink', $this->translate('Enable link to grafana'));
```

Comment:
I‘d suggest to keep permissions lowercase.

I would also do that for the config setting, but that‘s for a different issue.